script/Ciel/Day 5/05_01_CIEL05_4C.txt

```diff
@@ -74,7 +74,7 @@ I don't want to worry her, so I try to raise a hand to show her that I'm fine, b
 [sha:ed4e2df272653caab5fff593e6020ec60359d06c]{
 -- Page 770, Offset 12949.
 -- 呼吸―――呼吸はしていますね、心臓も動いている……なのに目を覚まさないなんて、ああ、もしかして頭が……！」
-He―――He's breathing. His heart is beating as well... But he isn't waking up, %{i}ahh%{/i}, maybe his head is...!"
+He―――He's breathing. His heart is beating as well... But he isn't waking up, ahh, maybe his head is...!"
```

Comment:
Why dropped?

Reply:
It's definitely a blurry distinction. I guess this one could go either way, but there's some ineffable point where these go from being "ah, yes" where 'ah' is a word to "_Aaahh_! Yes!" where it's more of an exclamation and should maybe be italicized.

Reply:
Hmm I see what you mean now. Alright I'll add it back